wordpress.org/public_html/wp-content/themes/pub/wporg-plugins-2024/template-parts/plugin-single.php

```diff
@@ -114,8 +120,13 @@
 			get_template_part( 'template-parts/section-advanced' );
 		} else {
 			$plugin_sections_titles = Template::get_plugin_section_titles();
+			$content_section_list = array( 'description', 'screenshots', 'blocks', 'installation', 'faq', 'reviews', 'developers', 'changelog' );
```

Comment:
Eek. This should be here, this was a failed understanding of how it worked in the beginning. Let me revert.